.github/CODEOWNERS

```
@@ -0,0 +1,4 @@
+# This file is used to define code owners for this repository.
+# See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+* @openaustralia/team-right-to-know
```

newline at end of file
```suggestion
* @openaustralia/team-right-to-know

```